native/src/main/java/io/ballerina/stdlib/http/api/service/signature/converter/JsonToRecordConverter.java

```diff
@@ -60,7 +67,14 @@ private static Object getRecordEntity(BObject entity, Type entityBodyType) {
      */
     private static Object getRecord(Type entityBodyType, Object bJson) {
         try {
-            return ValueUtils.convert(bJson, entityBodyType);
+            Map<String, Object> valueMap = new HashMap<>();
+            Boolean bool = Boolean.FALSE;
+            valueMap.put("enableConstraintValidation", bool);
```

Comment:
Define constants for this field names: `enableConstraintValidation` and `Options`